pumps/mongo_aggregate_test.go

```diff
@@ -101,7 +101,7 @@ func TestDoAggregatedWritingWithIgnoredAggregations(t *testing.T) {
 			//we build the query using the timestamp as we do in aggregated analytics
 			query := bson.M{
 				"orgid":     "123",
-				"timestamp": time.Date(timeNow.Year(), timeNow.Month(), timeNow.Day(), timeNow.Hour(), 0, 0, 0, timeNow.Location()),
+				"timestamp": time.Date(timeNow.Year(), timeNow.Month(), timeNow.Day(), timeNow.Hour(), timeNow.Minute(), 0, 0, timeNow.Location()),
```

Comment:
@mativm02 why are we changing this test?

Reply:
Let me explain it with an example:
What was happening before is that if the aggregated time was set to 60 minutes, and the current time was 10:13 am, the document timestamp would be set to 10:00. The timestamp of the next document would be 11:00 am. 
Now, if the aggregated time is 60 minutes and the current time is 10:13 am, the document timestamp will be set to 10:13 am. The timestamp of the next document will be 11:13 am.

Reply:
@mativm02 by default, we should maintain the existing behavior, so we don't break backward compatibility with the current pump users when they upgrade the version

Reply:
@tbuchaillot okay! now if the aggregation time is set to 60, it will create a new document every o'clock hour